src/containers/ScanBoxDetails.js

```diff
@@ -28,6 +28,7 @@ export const getMedian = (numbers) => {
 };
 
 const ScanBoxDetails = ({ ...props }) => (<div className="scanbox-details">
+  {console.log(props)}
```

Comment:
You have left console.log in code

Reply:
linter should catch this
@MostafaBalata can you check linter settings in the ico-contracts and update? we have "no-console" setting

Reply:
I disabled it, because we are using it to log the first block number and last block number when we add new ICO.